.jscsrc

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
Sadly no, there’s a fair bit of differences from JS in the browser, to client-side JS, and then JS on the server. Plus different teams do different things and changing now would be a PITA.

I think the client-side stuff is kinda consistent as we (old guard webdev) wrote up things in the webdev bootcamp about it (http://mozweb.readthedocs.org/en/latest/reference/js-style.html), but that might even be a bit out-of-date by now...